scripts/game_structure/windows.py

```diff
@@ -670,6 +669,10 @@ def process_event(self, event):
                 game.all_screens["profile screen"].exit_screen()
                 game.all_screens["profile screen"].screen_switches()
                 self.kill()
+        elif event.type == pygame.KEYDOWN and event.key == pygame.K_ESCAPE and game.settings["keybinds"]:
+            game.all_screens["profile screen"].exit_screen()
```

Comment:
Don't exit the background screen, just close the menu. In other words, if you have this menu open and press ESC, close this menu *and nothing else*. Sorry I haven't been super clear on that 😅